yt/visualization/tests/test_callbacks.py

```diff
@@ -342,6 +345,7 @@ def test_text_callback():
                         text_args={'color':'red'})
         assert_fname(p.save(prefix)[0])
 
+@requires_ds(cyl_2d)
```

Comment:
This should be `requires_file` because these aren't answer tests. You also need to import both `requires_file` and `yt`. Check flake8.